File: FE/WEB/app/models/translation.js
import Model, { attr } from '@ember-data/model';

export default class TranslationModel extends Model {
  @attr('string', { defaultValue: 'it' }) languageCode;
  @attr('arrays', { defaultValue: '{}' }) translationWeb;
  @attr('arrays', { defaultValue: '{}' }) translationApp;
}

File: BE/Src/MIT.Fwk.Infrastructure/Entities/Translation.cs
using JsonApiDotNetCore.Resources;
using JsonApiDotNetCore.Resources.Annotations;
using MIT.Fwk.Core.Attributes;

namespace MIT.Fwk.Infrastructure.Entities
{
    [Resource]
    [SkipJwtAuthentication(JwtHttpMethod.GET)]
    public class Translation : Identifiable<int>
    {
        [Attr]
        public string languageCode { get; set; }

        [Attr]
        public string translationWeb { get; set; }

        [Attr]
        public string translationApp { get; set; }


    }
}
